ide/static/ide/js/compile.js

```diff
@@ -335,9 +335,34 @@ CloudPebble.Compile = (function() {
         }
     };
 
+    function logs_scrolled_to_bottom() {
+        if (mLogHolder == null) {
+            return false;
+        }
+        return (mLogHolder[0].scrollHeight - mLogHolder.scrollTop() == mLogHolder.outerHeight());
```

Comment:
You should leave a few pixels of error here; otherwise it's too easy to think you're at the bottom and then be left behind.